src/service_wrapper.cpp

```diff
@@ -285,3 +288,7 @@ void ServiceWrapper::parameterModificationResponseReceived(const rclcpp::Client<
 
     responseQueue.push(response);
 }
+
+void ServiceWrapper::setIgnoreDefaultParameters(bool ignoreDefaultParameters_) {
+    this->ignoreDefaultParameters=ignoreDefaultParameters_;
```

Comment:
```suggestion
    this->ignoreDefaultParameters = ignoreDefaultParameters_;
```
small personal preference :D